buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -149,11 +149,6 @@ class AirbytePythonPlugin implements Plugin<Project> {
                 command = "install -r requirements.txt"
                 inputs.file('requirements.txt')
                 outputs.file('build/installedlocalreqs.txt')
-
-                // HACK: makes all integrations depend on installing requirements for bases. long term we should resolve deps and install in order.
-                if (project.getPath().startsWith(":airbyte-integrations:connectors") && !project.hasProperty("reports_folder")) {
-                    dependsOn project.rootProject.getTasksByName("airbytePythonApply", true).findAll { it.project.getPath().startsWith(":airbyte-integrations:bases") }
-                }
```

Comment:
Sherif knows more than me here so I want him to confirm.

My thoughts:
I don't think we can remove this without changing more dependencies. This block basically makes each connector submodule depend on all the submodules in the bases folders e.g. (base, base-java, base-java-s3), this is required for connectors whose docker file depends on a `dev` base image. See [this](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-bigquery/Dockerfile#L1) for example.

To remove this, I think we would need to go through all the connector dockerfiles and make sure they either use a versioned base image so we don't have any local image expectations.

Reply:
What's the risk factor here of pushing ahead and addressing issues as they come up?

This is a massive improvement that will undoubtedly save us a not-insignificant amount of time and $$ on ci bills.

Reply:
@davinchia thanks for your reply.
I'm under the impression this "hack" is safe to remove because:
1. It's python only. As you mentioned, there could be more stickiness to `base` pre-installation on Java connector, but this change is python connector specific.
2. It's always formatting and installing base-normalization and connector-acceptance-test, even if these are not defined as a dependency on the connector side which I think is a superfluous task in the context of connector build.
3. We already have the guarantee that connector acceptance tests are installed in the connector because it is defined in their `requirement.txt` ([eg](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pokeapi/requirements.txt#L1)) and installed in the `installLocalReqs` task.
I don't know exactly why we make connector-acceptance-test a test requirements dependency of connectors and why we declare this dependency both in `setup.py` and `requirements.txt`.

@sherifnada please share I you have more details on this matter 😄 🙏

Reply:
@alafanechere I missed that this is only for the python modules. If that's true, I believe your comment is correct. I double checked by spot checking a couple of python connector build setups - don't seem like they directly rely on the various connector bases. 

Have you tried removing this and build 2 - 3 python connectors? That's the best sanity check.